Radio Downloader.csproj

```diff
@@ -37,10 +40,10 @@
     <MapFileExtensions>true</MapFileExtensions>
     <ApplicationRevision>1</ApplicationRevision>
     <ApplicationVersion>1.0.0.%2a</ApplicationVersion>
-    <IsWebBootstrapper>false</IsWebBootstrapper>
     <UseApplicationTrust>false</UseApplicationTrust>
     <BootstrapperEnabled>false</BootstrapperEnabled>
-    <TargetFrameworkProfile />
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
```

Comment:
This cruft that Visual Studio has added can be dropped out of this PR.

Reply:
With this and a few comments below relating to 'Radio Downloader.csproj', you suggest reverting or dropping out content.  It this best done via a git command or by direct editing the file through visual studio?